code/datums/elements/delete_on_drop.dm

```diff
@@ -0,0 +1,20 @@
+/**
+ * Attaches to an item, if that item is dropped on the floor delete it
+ */
+/datum/element/delete_on_drop
```

Comment:
why did you add this theres an item flag that does exactly the same thing

Reply:
There is?